### DISCUSSION TITLE
Handle the case where localStorage is not available

### DISCUSSION THREAD

**Discussion by heliodor** [grafana/grafana#37220] [Dashboards, Panels & Visualizations]
Continuing the discussion here for this bug report:
https://github.com/grafana/grafana/issues/15884

This bug occurs if you embed a dashboard in an iframe, serve them from different domains, and disable third-party cookies:

![101706281-9ff18d80-3aed-11eb-8f1a-46c3d94d0a02](https://user-images.githubusercontent.com/395678/127028233-141236d5-17bb-4554-bf93-c459819a6f2a.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
